src/components/ExternalLink.vue

```diff
@@ -42,11 +42,20 @@ const formattedText = computed(() => {
         color: var(--link-color);
         display: inline-flex;
         align-items: center;
+        gap: 4px;
+        max-width: 100%;
+    }
+
+    &__text {
+        display: inline-block;
+        min-width: 0;
+        overflow: hidden;
+        white-space: nowrap;
+        text-overflow: ellipsis;
```

Comment:
this makes links which are too long ellipsize, like very long NFT collection names